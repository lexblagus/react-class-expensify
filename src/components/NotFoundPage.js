import React from 'react';
import {Link} from 'react-router-dom'

const NotFoundPage = () => (
	<div className="hero is-danger">
		<h1 className="title is-1">Page not found.</h1>
		<p>go <Link to="/">home</Link></p>
	</div>
);

export default NotFoundPage;
