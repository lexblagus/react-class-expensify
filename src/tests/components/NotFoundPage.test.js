import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage  from '../../components/NotFoundPage';
import storeDummyData from '../fixtures/storeDummyData';


test('Should render NotFoundPage', ()=>{
	//console.log( {...storeDummyData[0]} );
	
	const wrapper = shallow(
		<NotFoundPage />
	);
	
	expect(wrapper).toMatchSnapshot();
});

