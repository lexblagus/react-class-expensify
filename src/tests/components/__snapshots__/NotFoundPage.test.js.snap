// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render NotFoundPage 1`] = `
<div
  className="hero is-danger"
>
  <h1
    className="title is-1"
  >
    Page not found.
  </h1>
  <p>
    go 
    <Link
      replace={false}
      to="/"
    >
      home
    </Link>
  </p>
</div>
`;
